portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,107 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+class Place {
+  constructor(name, lat, lng, year, message = "") {
+    this.name = name;
+    this.lat = lat;
+    this.lng = lng;
+    this.year = year;
+    this.message = message;
+  }
+}
+
+/** Places I have lived. */
+const placesLived = [
+  new Place('Evanston', 42.0451, -87.6877, '2000-2002', 'I was born in outside Chicago at Northwestern Medical Center.'),
+  new Place('Overland Park', 38.9822, -94.6708, '2002-2017', 'I grew up in the Kansas City suburbs, first in Lenexa and then Overland Park. I attended Blue Valley North high school for three years.'),
+  new Place('Wassenaar', 52.1429, 4.4012, '2017-2018', 'I moved to the Netherlands before my senior year because my father switched jobs. I graduated from the American School of the Hague.'),
+  new Place('Athens', 33.9519, -83.3576, '2018-Present', 'I now live in Athens, Georgia. I am going into my junior year at UGA.'),
+  new Place('Oxford', 51.7520, -1.2577, 'May-June 2019', 'For the beginning of summer after my freshman year, I studied Computing Ethics at Oxford through the Foundation Fellowship\'s Maymester Program')
+]
+
+/** Places I have visited. */
+const placesVisited = [
+  // Domestic
+  new Place('New York', 40.7128, -74.0060, 2019),
```

Comment:
[Optional] I think using a Builder pattern would make this code even cleaner.  If you aren't familiar you can check out Effective Java, Chapter 10, Item 2 (https://github.com/muhdkhokhar/test/blob/master/Joshua%20Bloch%20-%20Effective%20Java%20(3rd)%20-%202018.pdf)

Reply:
I tried using a Builder pattern but it was not letting me string together multiple methods (i.e. doing new Place.Builder('New York').withLatitude(40.7128).withLongitude(-74.0060) would throw an error) so I ultimately did not use it.

Reply:
I was able to implement a Builder pattern in the latest commit.